apis/management.cattle.io/v3/rke_types.go

```diff
@@ -4,6 +4,7 @@ import (
 	"github.com/rancher/norman/types"
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	apiserverconfig "k8s.io/apiserver/pkg/apis/config"
```

Comment:
@moelsayed import from the v1 version package instead (`apiserverconfig "k8s.io/apiserver/pkg/apis/config/v1")